### PR TITLE
Throws errors and exist when config encounters an error

### PR DESCRIPTION
**Pull request by good-idea**
Addresses #69 

### Description

Previously, the config loader would fall back to the default configuration when it encountered an error loading `doczrc.js`. This PR changes the behavior to report the error and fail instead.

### CODE REVIEW

**Comment by good-idea** [comment]
This also installs `libundler` as a devdependency - I don't like making global installs. I didn't add the `package-lock.json` that was generated - I wasn't sure if you had ignored this for a reason or not.

File: packages/load-cfg/src/index.ts [hunk 1]
```diff
@@ -43,7 +43,8 @@ export const loadFile = (
       file = JSON.parse(fs.readFileSync(filepath, 'utf-8'))
     }
   } catch (err) {
-    file = defaultFile
+    console.warn('There was an error loading your doczrc.js config:')
```
Comment (pedronauck) [inline packages/load-cfg/src/index.ts]: please, remove `doczrc.js` because `load-cfg` is a generic package not specific!

**Review by pedronauck — CHANGES_REQUESTED** [review]

File: packages/load-cfg/package.json [hunk 2]
```diff
@@ -27,6 +27,7 @@
   },
   "devDependencies": {
     "@types/find-up": "^2.1.1",
-    "@types/node": "^10.3.2"
+    "@types/node": "^10.3.2",
+    "libundler": "^1.7.1"
```
Comment (pedronauck) [inline packages/load-cfg/package.json]: I don't need to add `libundler` as dependency, we're using lerna. With lerna you can have `devDependencies` just in the root package! But you need to run `lerna bootstrap`

Comment (good-idea) [reply packages/load-cfg/package.json]: got it, thanks!

**Review by pedronauck — APPROVED** [review]

**Pull request merged**